packages/toolkit/package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "@babel/core": "^7.17.8",
     "@babel/eslint-parser": "^7.17.0",
-    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.5",
+    "@pmmmwh/react-refresh-webpack-plugin": "https://github.com/nicholasio/react-refresh-webpack-plugin.git",
```

Comment:
Is this fork related to this change? I'm mainly curious

Reply:
Yes - hopefully we can go back to upstream once this is merged and released https://github.com/pmmmwh/react-refresh-webpack-plugin/pull/726